app/js/preferences/preferences.js

```diff
@@ -35,7 +35,11 @@ define([
         'problem',
         'settings',
         'updateAngularScenario',
-        'toggledColumns'
+        'toggledColumns',
```

Comment:
What does the preferences have to do with toggled columns. As far as I know it there are no toggleable columns on this tab.

Reply:
Goes into Settings context?